base/logging.jl

```diff
@@ -274,10 +274,76 @@ end
 
 default_group(file) = Symbol(splitext(basename(file))[1])
 
+function issimple(@nospecialize val)
+    val isa String && return true
+    val isa Symbol && return true
```

Comment:
characters can also show up here.
```suggestion
    val isa Symbol && return true
    val isa Char && return true
```